homework_unit_8/menu/style.scss

```
@@ -0,0 +1,346 @@
+                        transform: none;
+                    }
+
+                    i {
+                        transform: none;
+                    }
+                }
+            }
+        }
+    }
+
+    .navbar__search {
+        display: flex;
+        align-items: center;
+        height: 35px;
+        position: relative;
+
+        .navbar__search__input {
+            height: calc( 100% - 2px);
+            width: 80%;
+            border: none;
+            outline: none;
+            padding: 0 11px;
+            border: 1px solid $color-border;
+        }
+
+        .navbar__search__btn {
+            background-color: $color-hover;
+            outline: none;
+            border: none;
+            color: $color-white;
+            padding: 0 20px;
+            line-height: 35px;
+            height: 100%;
+            position: absolute;
+            right: 0;
+        }
+    }
+
+    .navbar__dropdown {
+        position: absolute;
+        top: 0;
+        left: 100%;
+        width: 100%;
+        background-color: $color-white;
+        transition: opacity 0.2s linear, height 0.2s linear;
+        border-left: 6px solid $color-hover;
+        z-index: 1;
+        opacity: 0;
+
+        & > li {
+            padding: 15px 20px;
+            position: relative;
+            border-bottom: 1px solid $color-border;
+
+            & > a {
+                color: $color-gray;
+                font-size: 20px;
+                transition: all 0.1s linear;
+
+                & > i {
+                    margin-right: 15px;
+                    transition: transform 0.2s linear;
+                }
+
+                & > div {
+                    transition: transform 0.2s linear;
+                    display: inline-block;
+                }
+            }
+
+            &:hover > a{
+                color: $color-hover;
+
+                div {
+                    transform: translateX(10px);
+                }
+
+                i {
+                    transform: scale(1.2);
+                }
+            }
+
+            &:hover .navbar__dropdown__staff {
+                animation: fadeInDown 0.2s linear;
+                opacity: 1;
+            }
+        }
+        &::before {
+            content: "";
+            position: absolute;
+            top: 10%;
+            transform: translateY(-10%);
+            left: -22px;
+            border-width: 8px;
+            border-style: solid;
+            border-color: transparent $color-hover transparent transparent;
+            z-index: 99;
+        }
+
+        & .navbar__dropdown__staff {
+            position: absolute;
+            top: 0;
+            left: 100%;
+            width: 80%;
+            background-color: $color-white;
+            transition: opacity 0.2s linear, height 0.2s linear;
+            border-left: 6px solid $color-hover;
+            z-index: 2;
+            opacity: 0;
+
+            & > .navbar__dropdown__item {
```

```suggestion
            > .navbar__dropdown__item {
```
Xóa mấy dấu `&` vô lý đi